config/owasp/suppressions.xml

```diff
@@ -1,14 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
-  <suppress until="2030-01-01">
-    <notes><![CDATA[
-     Suppressing as it's a false positive
-   ]]></notes>
-    <gav regex="true">^org\.springframework\.security:spring-security-crypto:5.2.0.RELEASE</gav>
-    <cpe>cpe:/a:pivotal_software:spring_security</cpe>
-    <cve>CVE-2018-1258</cve>
-  </suppress>
-
```

Comment:
hmm. must be coming from cloud or where. which is not present at the moment iirc